test/core/tableTests.ts

```diff
@@ -68,14 +68,41 @@ describe("Tables", () => {
     assert.isNull(rows[1][0], "component at (1, 0) is null");
   });
 
-  it("can't add a component where one already exists", () => {
+  it("Add a component where one already exists creates a new group", () => {
     var c1 = new Plottable.Component.AbstractComponent();
     var c2 = new Plottable.Component.AbstractComponent();
     var c3 = new Plottable.Component.AbstractComponent();
     var t = new Plottable.Component.Table();
+
     t.addComponent(0, 2, c1);
     t.addComponent(0, 0, c2);
-    assert.throws(() => t.addComponent(0, 2, c3), Error, "component already exists");
+    t.addComponent(0, 2, c3);
+
+    assert.isTrue(Plottable.Component.Group.prototype.isPrototypeOf((<any> t)._rows[0][2]));
+
+    var components: Plottable.Component.AbstractComponent[] = (<any> t)._rows[0][2].components();
+    assert.lengthOf(components, 2, "The group created should have 2 components");
+    assert.equal(components[0], c1, "First element in the group at (0, 2) should be c1");
+    assert.equal(components[1], c3, "Second element in the group at (0, 2) should be c3");
+  });
+
+  it("Add a component where a group already exists adds the component to the group", () => {
+    var c1 = new Plottable.Component.AbstractComponent();
+    var c2 = new Plottable.Component.AbstractComponent();
+    var c3 = new Plottable.Component.AbstractComponent();
+    var t = new Plottable.Component.Table();
+
+    t.addComponent(0, 2, c1);
+    t.addComponent(0, 2, c2);
+    t.addComponent(0, 2, c3);
```

Comment:
I'd prefer that instead of doing this, you create a `Group` and add it to the cell. This makes this test independent of the previous one (which checks for `Group` creation on collision).